samples/dnn/models.yml

```diff
@@ -49,6 +49,20 @@ yolov4-tiny:
   classes: "object_detection_classes_yolov4.txt"
   sample: "object_detection"
 
+yolov3:
+  load_info:
+    url: "https://pjreddie.com/media/files/yolov3.weights"
+    sha1: "520878f12e97cf820529daea502acca380f1cb8e"
+  model: "yolov3.weights"
+  config: "yolov3.cfg"
+  mean: [0, 0, 0]
+  scale: 0.00392
+  width: 416
+  height: 416
+  rgb: true
+  classes: "object_detection_classes_yolov3.txt"
```

Comment:
This file does not exist

Reply:
Sorry, @dkurt ! I forgot... done and thanks.